mysql-test/suite/tokudb.perfschema/r/crash_tokudb.result

```diff
@@ -0,0 +1,30 @@
+SELECT COUNT(*) > 0 FROM performance_schema.setup_consumers
```

Comment:
tokudb.perfschema is a new MTR suite, right? If so, it has to be added to the default suite list in mysql-test-run.pl